### DISCUSSION TITLE
Day 2

### DISCUSSION THREAD

**Discussion by chiche-ds** [chiche-ds/30-Day-Terraform-challenge-#16] [Day-to-Day Tasks]
Day 2 task is ready 
things are beginning to get serious 
we are reading chapter 2 and watching more videos

**Comment by kevshakes** [comment]
> Day 2 task is ready 
> things are beginning to get serious 
> we are reading chapter 2 and watching more videos 

Thank you @chiche-ds

**Comment by Assan-zidan** [comment]
Let's start chapter 2
